csrc/ops/arith.h

```diff
@@ -452,12 +452,6 @@ NVF_API Val* atan2(Val* v1, Val* v2);
 NVF_API TensorView* atan2(TensorView* v1, Val* v2);
 NVF_API TensorView* atan2(Val* v1, TensorView* v2);
 NVF_API TensorView* atan2(TensorView* v1, TensorView* v2);
-// truediv: promote to float for integer division, has the same semantics as the
-// python's operator /
-NVF_API Val* truediv(Val* v1, Val* v2);
-NVF_API TensorView* truediv(TensorView* v1, Val* v2);
-NVF_API TensorView* truediv(Val* v1, TensorView* v2);
-NVF_API TensorView* truediv(TensorView* v1, TensorView* v2);
```

Comment:
Naive question, we don't think we need/should remove the c++ truediv API. Since these does float promotion, so it is a truediv in nature? Even though it's not a BinaryOpType::TrueDiv.